test/testlib/nuodb_admin_utilities.go

```diff
@@ -303,3 +303,22 @@ func AwaitServerState(t *testing.T, namespace string, adminPod string,
 		return false
 	}, timeout)
 }
+
+// VerifyLables checks if adminPod in namespace has label and its value is one of the expectedValues.
+func VerifyLables(t *testing.T, namespace string, adminPod string, label string, expectedValues []string) {
```

Comment:
We always know exactly what value the label is supposed to have, so why do we take an array?

Reply:
We will have multiple values for node (and possibly other labels) if the test is run on a multi-node cluster.

Reply:
Even if we ran these tests on a multi-host cluster like Kind, at the point where this is invoked in the test, the AP has already been scheduled on a node, so we would know exactly what labels it's supposed to have since we can get them from the node.

Reply:
So we would need to get the node name from the AP and then look up the labels for that Node. It is more work, but I suppose that it does check that the AP is not picking the labels from the wrong node...